README.md

```diff
@@ -133,6 +133,14 @@ You can include the library as a regular script tag on your page:
 
 If you are importing the Auth module, you do not need to import the standard SDK module (`dist/onfido.min.js`) also.
 
+Alternatively, you can use hosted versions of files above from our CDN such as:
+
+```html
+<script src="https://assets.onfido.com/web-sdk-releases/8.3.0/onfido.min.js"></script>
```

Comment:
It looks like https://assets.onfido.com/web-sdk-releases/latest/onfido.min.js works as well. I'd recommend we leave `latest` in the example, and add a note saying we can replace `latest` with any version number. Otherwise, people will just copy-paste the example and we will forget to update it at each release.